traveler-ios-ui/TravelerKitUI/Views/WishlistCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  WishlistCell.swift
```

Comment:
Cells shouldn't be aware of where they are being implemented because it makes them less reusable. Find a more generic term for a cell that would contain this kind of information so that re usability is encouraged. Perhaps you might want try something like "AvailabilityCell"